### ISSUE TITLE
Support for dtmf

### ISSUE THREAD

**Issue by aososam** [gdtrudian/td_sip_plugin#2]
When on a SIP call when someone presses a number there is no way to get this action using this library. is it possible to add?

**Comment by ghost** [comment]
Yes, this is really important function 👍 that We are also looking for.

**Comment by ghost** [comment]
Hi, @gdtrudian, 
we are ready to pay for this feature for us (https://ttc-marconi.com/en/)
Can you help us with this issue? 

Thank you 
Tomas
